src/oauth2/client.cr

```diff
@@ -106,42 +106,54 @@ class OAuth2::Client
 
   # Gets an access token using an authorization code, as specified by
   # [RFC 6749, Section 4.1.1](https://tools.ietf.org/html/rfc6749#section-4.1.3).
```

Comment:
What about also fixing this link? In the text it says "Section 4.1.1" but in the URL it says "section-4.1.3".